base/commands/viridian/viridian_cluster_create.go

```diff
@@ -0,0 +1,105 @@
+package viridian
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/hazelcast/hazelcast-commandline-client/clc"
+	. "github.com/hazelcast/hazelcast-commandline-client/internal/check"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/output"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/plug"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/serialization"
+	"github.com/hazelcast/hazelcast-commandline-client/internal/viridian"
+)
+
+type ClusterCreateCmd struct{}
+
+func (cm ClusterCreateCmd) Init(cc plug.InitContext) error {
+	cc.SetCommandUsage("create-cluster [flags]")
+	long := `Creates a Viridian cluster for the logged in API key.
+
+Make sure you login before running this command.
+`
+	short := "Creates Viridian cluster"
+	cc.SetCommandHelp(long, short)
+	cc.SetPositionalArgCount(0, 0)
+	cc.AddStringFlag(propAPIKey, "", "", false, "Viridian API Key")
+	cc.AddStringFlag(flagName, "", "", false, "override the cluster name")
```

Comment:
Maybe instead of "override the cluster name": "specify the cluster name; if not given an auto-generated name is used."